### PR TITLE
Installer: add periodics to test OKD using FCOS stable/testing for firstboot

### PR DESCRIPTION
**Pull request by vrutkovs**
This PR adds a step to patch installer with FCOS AMI from selected stream and adds two periodic jobs to check OKD 4.12 install on FCOS testing stream and OKD 4.11 on stable stream

### CODE REVIEW

**Comment by vrutkovs** [comment]
okd-e2e-aws-ovn-stable-stream passed, testing stuck on bootstrap
/retest

**Comment by cgwalters** [comment]
Seems sane to me, though it's worth noting that this will only test the firstboot flow - the running OS will still be replaced with the target OS content.  I assume that's intentional here, but I think it's worth spelling out for anyone later who might be confused at why e.g. the `kernel` in the target stream isn't used when kubelet is running.

**Comment by vrutkovs** [comment]
>this will only test the firstboot flow - the running OS will still be replaced with the target OS content

Correct. For latter we're going to need a more elaborate promotion periodic

File: ci-operator/step-registry/ipi/conf/aws/fcos-ami/ipi-conf-aws-fcos-ami-commands.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+# Install tools
+echo "## Install yq"
+curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+echo "   yq installed"
+
+echo "## Install jq"
+curl -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 -o /tmp/jq && chmod +x /tmp/jq
+echo "   jq installed"
```
Comment (patrickdillon) [inline ci-operator/step-registry/ipi/conf/aws/fcos-ami/ipi-conf-aws-fcos-ami-commands.sh:16]: These are now baked into the upi image... can you use those instead?

**Pull request merged**